.circleci/config.yml

```diff
@@ -9,10 +9,18 @@ jobs:
       - image: circleci/node:7.10
     steps:
       - checkout
-      - run: cd client && yarn install && yarn test
+      - run: cd client && ./bin/ci_running.sh && yarn lint
+      - run: cd client && ./bin/ci_running.sh && yarn test
+  functions_test:
+    docker:
+      - image: circleci/node:7.10
```

Comment:
not compatible with js version of functions right ?

Reply:
Done